src/tv2-common/helpers/graphics/caspar/index.ts

```diff
@@ -165,3 +165,146 @@ function HTMLTemplateContent(config: TV2BlueprintConfig, graphicTemplate: string
 		}
 	}
 }
+
+export function getHtmlGraphicBaseline(config: TV2BlueprintConfig) {
+	const templateName = getHtmlTemplateName(config)
+	return [
+		...getSlotBaselineTimelineObjects(templateName),
+		getCompoundSlotBaselineTimelineObject(templateName),
+		getDesignBaselineTimelineObject(templateName),
+		getPilotBaselineTimelineObject(templateName),
+	]
+}
+
+function getSlotBaselineTimelineObjects(templateName: string): TSR.TSRTimelineObj[] {
+	const slotBaselineObjects: TSR.TSRTimelineObj[] = []
+	;[
+		SharedGraphicLLayer.GraphicLLayerOverlayIdent,
+		SharedGraphicLLayer.GraphicLLayerOverlayLower,
+		SharedGraphicLLayer.GraphicLLayerOverlayTema,
+		SharedGraphicLLayer.GraphicLLayerOverlayTopt,
+		SharedGraphicLLayer.GraphicLLayerLocators
+	].forEach(layer => {
+		if (layerToHTMLGraphicSlot[layer]) {
+			slotBaselineObjects.push(
```

Comment:
This is kinda creating a 'side effect' of your foreach. 
In general, you should be affecting as little as possible outside of your current scope and since `slotBaselineObjects` is declared outside of your foreach, it is out of scope here. (Sidenote: foreach should most just be used for a quick iteration of a list that has minimal computation i.e. console logging each entry)

Instead you can utilize `.map()`, so it could look something like this:
```
const slotBAselineObject: TSR.TSRTImelineObj[] = [
  SharedGraphicLLayer.GraphichLLayerOverlayIdent,
  // Rest of the layers
].filter(layer => layerToHTMLGraphicSlot[layer])
.map(layer => {
  return // your timelineObject
})
```

Reply:
But I know see you just moved the code.
Would be nice with some boyscouting :)